src/http-schema/http-schema.ts

```diff
@@ -75,12 +75,12 @@ const ImageObjectContent = Joi.object({
 
 // These are not imported yet
 const OtherContent = Joi.object({
-  type: Joi.string().valid('CodeBlock', 'MathFragment', 'MediaObject', 'Table', 'ThematicBreak'),
+  type: Joi.string().valid('CodeBlock', 'MathFragment', 'MediaObject', 'Table', 'ThematicBreak', 'List'),
 });
 // end block
 
 const ContentPartSchema = Joi.alternatives().try(
-  Joi.string(),
+  Joi.string().allow(null, ''),
```

Comment:
Should we allow empty content blocks? This seems like something we should filter out if they are providing no usable content or at least flag a warning that this should be investigated in case there is content missing.